### PR TITLE
Feature/add json validation against json schema #18

### PR DESCRIPTION
**Pull request by 4lm**
Hi all,

This is the first draft of a metadata schema with descriptions, as discussed in #18. 

Please review the schema definition of v1.4 (the second one). v1.3 will follow as soon as I have access to the descriptions.

Please note, I used `TBD` to mark descriptions which need to be decided on, either because they are undefined or they contain mistakes.

Edit: I also added `$id` and `$schema`.

### CODE REVIEW

**Comment by 4lm** [comment]
Please note, I forgot to mark this PR as a "draft" PR. Seems like, I can't change this flag now, so please note this PR is not for merging (yet).

**Comment by Bachibouzouk** [comment]
Nice work @4lm !
Can you test locally that the old schema (without the description) and the new have the same schema (i.e. that you did not include a comma in a string or skipped a `}` or a `{`

We could add in the tests folder that the example `.json` file should match the corresponding schemas.

Don't forget to update the CHANGELOG at the end :)

**Comment by 4lm** [comment]
@Bachibouzouk. I just added two simple test for the two JSON schemas. Just run `tox` in the console. Does this answer your question?

File: tests/test_schemas.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
+import json
+
+import jsonschema
+import pytest
+
+from omi.schemas import METADATA_SCHEMA_v1_3_0
+from omi.schemas import METADATA_SCHEMA_v1_4_0
+
+METADATA_EXAMPLE_v1_3_0 = json.loads(
```
[the remark this answers is not in the document]
Comment (4lm) [reply tests/test_schemas.py]: > Can't these examples be located somewhere else (right now they are available here? 

Mhh, v1.3 is the "template" and v1.4 the (modified: null -> "null") "example" from the [examples](https://github.com/OpenEnergyPlatform/examples/tree/master/metadata) - I think I got confused :wink:. I will correct this and use both time the examples. The examples section in [examples](https://github.com/OpenEnergyPlatform/examples/tree/master/metadata) is not very clean. For example I don't understand why there are unversioned (filenames) examples and templates and versioned examples and templates in a subfolder called archiv. I will clean this up and do a pull request.

> That was part of this discussion. I would be in favor of having METADATA_TEMPLATE_v1_x_0 somewhere inside the omi repo, then we can also import it for our test.

I know, hopefully we have a decision what to put where soonish (@MGlauer, @Ludee and @christian-rli are meeting in Freiburg, so there is a decision maybe after that meeting). I also requested them to talk about this: https://github.com/OpenEnergyPlatform/organisation/issues/26#issuecomment-545397922 - let's see if they find the time for a decision. Until then, please see the code of this draft PR as temporal/provisional. For me right now is more important to talk about the actual schemas. I will implement the description of v1.3 today.

**Comment by 4lm** [comment]
Will close this PR. See here why: https://github.com/OpenEnergyPlatform/organisation/issues/26#issuecomment-548330375

**Pull request closed without merging**